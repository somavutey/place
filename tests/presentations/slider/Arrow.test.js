import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Arrow from "../../../components/presentations/Slider/Arrow";

it("Arrow", () => {
  const wrapper = shallow(<Arrow />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
