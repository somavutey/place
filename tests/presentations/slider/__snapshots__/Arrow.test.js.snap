// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow 1`] = `
<styled.div>
  <img
    alt="right arrow"
    className="h-6 w-6"
  />
</styled.div>
`;
